From 0503ec88e9187c0152b7b2840a1ad5bfb022bbfe Mon Sep 17 00:00:00 2001
From: Colin Ian King <colin.i.king@gmail.com>
Date: Mon, 1 Aug 2022 21:28:49 +0100
Subject: [PATCH 2/2] core-shim: remove include of sys/mount.h

This is not required in the shim core and it fixes a build issue
with newer glibc 2.36

Fixes: https://github.com/ColinIanKing/stress-ng/issues/216

Upstream-Status: Backport [https://github.com/ColinIanKing/stress-ng/commit/0c9a711f213b5734729ab0c5ed90669e9fd11ca2]
Signed-off-by: Colin Ian King <colin.i.king@gmail.com>
---
 core-shim.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/core-shim.c b/core-shim.c
index 0343402a..324eba7d 100644
--- a/core-shim.c
+++ b/core-shim.c
@@ -52,10 +52,6 @@
 #include <asm/ldt.h>
 #endif
 
-#if defined(HAVE_SYS_MOUNT_H)
-#include <sys/mount.h>
-#endif
-
 #if defined(HAVE_SYS_PRCTL_H)
 #include <sys/prctl.h>
 #endif
-- 
2.37.1

